src/inputProcessing/inputs.ts

```diff
@@ -28,6 +30,7 @@ export function readInputs() {
   const assignmentsInput = core.getInput(INPUT_ASSIGNMENTS_KEY);
   const enforcementModeInput = core.getInput(INPUT_ENFORCEMENT_MODE_KEY);
   mode = core.getInput(INPUT_MODE) ? core.getInput(INPUT_MODE).toLowerCase() : MODE_INCREMENTAL;
+  forceUpdate = core.getInput(FORCE_UPDATE_KEY) ? core.getInput(FORCE_UPDATE_KEY).toLowerCase() == "true" : false;
```

Comment:
Lets set this to hard coded FALSE till we enable in action.yml

Reply:
We can have it as it is.